ramls/item.json

```
@@ -39,6 +39,10 @@
       "type": "string",
       "description": "Unique inventory control number for physical resources, used largely for circulation purposes"
     },
+    "shelvingOrder": {
```

As this is a derived property, I think we should follow the convention and use the `effective` prefix.

```suggestion
    "effectiveShelvingOrder": {
```

---

Does it mean that we need new object "shelvingOrder" with single field "effectiveShelvingOrder" or field "effectiveShelvingOrder" should be placed in effective attributes block? In this case may be "itemLevelShelvingOrder" should be added too?

---

> Does it mean that we need new object "shelvingOrder" with single field "effectiveShelvingOrder" or field "effectiveShelvingOrder" should be placed in effective attributes block?

I only mean renaming the property that you have defined.

I don't think there needs to be a separate object. I believe the [domain folks have expressed](https://issues.folio.org/browse/MODINVSTOR-381?focusedCommentId=97850&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-97850) that the shelving order is not part of the call number components, so the property should not belong there.

Does that make sense?

> In this case may be "itemLevelShelvingOrder" should be added too?

I don't think there is a need for this prefix, as I don't believe there is the idea of a shelving order at a different level.

(I don't really know why that prefix was used in the previous work)

---

Have you already made this change?

---

It looks like this :
 "effectiveShelvingOrder": {
      "type": "string",
      "description": "A system generated normalization of the call number that allows for call number sorting in reports and search results",
      "readonly": true
    },
    "itemLevelCallNumber": {
      "type": "string",
      "description": "Call Number is an identifier assigned to an item, usually printed on a label attached to the item. The call number is used to determine the items physical position in a shelving sequence, e.g. K1 .M44. The Item level call number, is the call number on item level."
    },